src/concuerror_callback.erl

```diff
@@ -2172,7 +2252,12 @@ explain_error({unsupported_request, Name, Type}) ->
   io_lib:format(
     "A process sent a request of type '~w' to ~p. Concuerror does not yet support"
     " this type of request to this process.",
-    [Type, Name]).
+    [Type, Name]);
+explain_error({process_limit_exceeded, NumberOfProcesses}) ->
+  io_lib:format(
+    "The process limit of ~.10B processes has been exceeded. You should try and increase "
+    " the maximum number of total processes generated by using the option -n.",
+    [NumberOfProcesses]).
```

Comment:
Rephrase: 'Your test is using more than ~w processes (--\<long name\>). Increase the limit or consider using fewer processes.'